homeassistant/components/modbus/modbus.py

```diff
@@ -419,9 +419,15 @@ def pb_call(
         except ModbusException as exception_error:
             self._log_error(str(exception_error))
             return None
+        if not result:
+            self._log_error("Error: pymodbus returned None")
+            return None
         if not hasattr(result, entry.attr):
             self._log_error(str(result))
             return None
+        if not result:
+            self._log_error("Error: pymodbus returned isError True")
+            return None
```

Comment:
Is the second one needed?
```suggestion
```

Reply:
it misses a .isError() I was outtrixed by our commit check...and you reviewed faster than I could correct it.

Will be solved in a short moment (just need to fight mypy).